src/System.Net.Sockets/src/System/Net/Sockets/AcceptOverlappedAsyncResult.Windows.cs

```diff
@@ -58,7 +58,7 @@ internal override object PostCompletion(int numBytes)
                         SocketOptionLevel.Socket,
                         SocketOptionName.UpdateAcceptContext,
                         ref handle,
-                        Marshal.SizeOf(handle));
+                        IntPtr.Size);
```

Comment:
@jkotas, could you log a bug against Roslyn on this, at the very least to get an official stance on the behavior? FYI: @dotnet/roslyn-compiler